tilelang/language/builtin.py

```
@@ -709,6 +709,20 @@ def sync_threads(barrier_id: int = None, arrive_count: int = None):
     return tir.call_intrin("int32", "tir.tvm_storage_sync", "shared", *args)
 
 
+def sync_warp(mask: int = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)